internal/cli/cmd/login/login.go

```diff
@@ -128,24 +124,113 @@ func (o *option) authAndSetToken() error {
 	return nil
 }
 
-func (o *option) readAndSetCredentialInput() error {
-	if len(o.username) == 0 {
-		_, _ = fmt.Fprint(o.writer, "Username: ")
-		user, err := read.FromStdin()
-		if err != nil {
-			return err
+type MyWriteCloser struct {
```

Comment:
This is not ideal. 
There is an issue about that in promptui...
https://github.com/manifoldco/promptui/issues/211

Now reviewing the project and the lack of maintenance. I probably need to change my lib...

Reply:
used huh instead